README.md

```diff
@@ -28,30 +29,41 @@ To install vCluster Rancher Operator, you can add the loft repository to Rancher
 7. Follow the installation process and install the chart.
 8. In the sidebar, navigate to "Workloads" -> "Deployments". Confirm that the deployment named "vcluster-rancher-op" has the State "Active".
 
-Once the operator is installed, all vClusters deployed in any downstream cluster in rancher will cause a corresponding Rancher cluster to be created, the vCluster to connect to the corresponding Rancher cluster, and cluster owners added.
+Once the operator is installed:
+* All vClusters deployed in any downstream cluster will cause a corresponding Rancher cluster to be created
+* The vCluster will connect to the corresponding Rancher cluster
+* Any project-member or above will be added as a cluster-owner
 
 ## Uninstall
+
 1. Select the local cluster in the Rancher clusters overview page.
 2. In the sidebar, navigate to "Apps" -> "Installed Apps".
-3. Delete the vcluster-rancher-op app. The app will be have the name you gave it during install.
+3. Delete the vcluster-rancher-op app
+
+Note that Rancher clusters will not be affected by removing this operator.
 
 ## Technical Details
 
-The vCluster Rancher operator run in the local cluster and watches clusters.management.cattle.io. When a cluster has not been seen by the handler before,
-it creates a client using the Rancher reverse proxy to talk to the downstream cluster. This is done by creating a token, if needed, for the user.management.cattle.io
+
+*Downstream Client*
+
+The vCluster Rancher operator runs in Rancher's local cluster and watches clusters.management.cattle.io. 
+When a cluster has not been seen by the handler before, it creates a client using the Rancher reverse proxy to talk to the downstream cluster. This is done by creating a token, if needed, for the user.management.cattle.io resource
 that is created during the Helm chart install. The token is then used to authenticate requests to the Rancher proxy.
```

Comment:
This has changed a bit. The token is created on startup instead and every operation form that moment on uses a client build with that token instead of using the service account.

Reply:
I'll just remove that part about tokens, or simplify it.